setup.py

```diff
@@ -43,7 +43,7 @@
     extras_require={
         "dev": dev_requirements,
     },
-    url="https://github.com/hydrologie/xhydro",
-    version="0.1.5",
+    url="https://github.com/TC-FF/xhydro",
```

Comment:
You sure about this change?

Reply:
Yeah, I'm not sure what happened with this PR yesterday. Maybe because I locally merged our changes in `main`, which weren't yet in `Class-Data-in-frequency_analysis.local`? It tried to pull in all of our recent changes into this branch and the files changed were a mess.

Changing the base branch to `main` seems to have resolved it for now, although we now lose the comparisons with `Class-Data-in-frequency_analysis.local`. Given that mostly everything was rewritten, this is maybe not a big issue.